packet/Manager.py

```diff
@@ -244,3 +259,376 @@ def validate_capacity(self, servers):
     def get_spot_market_prices(self, params={}):
         data = self.call_api("/market/spot/prices", params=params)
         return data["spot_market_prices"]
+
```

Comment:
This is a lot of new functionality that works only in theory. An API SDK should contain Integration tests, at least for the most common use-cases. I mean tests that call the API.

Reply:
`https://github.com/TheAntColony/packet-python/blob/master/test/test_device.py#L40-L47` We tried to test where it was possible. BGP Configs were not tested, so we can either leave them or remove them.